compiler/luci-micro/CMakeLists.txt

```diff
@@ -35,6 +35,19 @@ FlatBuffers_Target(circle_schema
         SCHEMA_FILES "schema.fbs"
         )
 
+set(READER_SOURCE
+        "${CMAKE_CURRENT_SOURCE_DIR}/luci-interpreter/include/luci_interpreter/core/CircleMicroReader.h"
+        "${CMAKE_CURRENT_SOURCE_DIR}/luci-interpreter/include/luci_interpreter/core/CircleMicroReaderHelper.h"
+        "${CMAKE_CURRENT_SOURCE_DIR}/luci-interpreter/src/core/CircleMicroReader.cpp"
+        "${CMAKE_CURRENT_SOURCE_DIR}/luci-interpreter/src/core/CircleMicroReaderHelper.cpp"
+        )
+
+add_library(circle_reader STATIC ${READER_SOURCE})
```

Comment:
About the name `circle_reader`:
- I've found that `circle_reader` already landed
- This name looks general and can be assumed to be used in general
- Usually `circle_xxx` is used for tools not libraries

I strongly recommand to rename this target for luci-micro.